ql/src/test/results/clientpositive/llap/join_1to1.q.out

```diff
@@ -373,9 +373,6 @@ NULL	NULL	NULL	25	10025	66
 NULL	NULL	NULL	30	10030	88
 NULL	NULL	NULL	35	10035	88
 NULL	NULL	NULL	40	10040	88
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	66
-NULL	NULL	NULL	50	10050	88
```

Comment:
Indeed these rows shouldn't be there. It's a pity that people update q.out files without paying attention if the resulting changes are correct. We could have found this bug much earlier if people verified the results against another DBMS as well.

Reply:
yes; overwrite without checking tends to happen - that's why I see no value in the tests which use the `src` table - which content is not defined at all - one way around these problems is to use `assert_true` ...

Reply:
I am not a big fan of the the `src` and similar tables but if people validated the results against another DBMS I can leave with it.